run-testnet-relay.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+DONE_START='\n\e[1;32m' # new line + bold + green
+DONE_END='\n\n\e[0m'    # new line + reset
+
+WORKDIR=$PWD
+
+printf "${LOG_START}Initializing relay...${LOG_END}"
+
+cd tbtc/solidity
+
+# Get network ID.
+NETWORK_ID_OUTPUT=$(truffle exec ./scripts/get-network-id.js)
+NETWORK_ID=$(echo "$NETWORK_ID_OUTPUT" | tail -1)
+
+# Extract TestnetRelay contract address.
+JSON_QUERY=".networks.\"${NETWORK_ID}\".address"
+TESTNET_RELAY_CONTRACT="$WORKDIR/tbtc/solidity/build/contracts/TestnetRelay.json"
+TESTNET_RELAY_CONTRACT_ADDRESS=$(cat ${TESTNET_RELAY_CONTRACT} | jq "${JSON_QUERY}" | tr -d '"')
+
+printf "${LOG_START}TestnetRelay contract address is: ${TESTNET_RELAY_CONTRACT_ADDRESS}${LOG_END}"
+
+cd $WORKDIR
+
+# Copy all config files to the right relay directory.
+cp -R configs/relays/. relays/maintainer/maintainer/config/
+
+cd relays/maintainer/maintainer/config
+
+# Fill SUMMA_RELAY_CONTRACT env in config file with their actual value.
+sed -i .OLD 's:RELAYCONTRACT:'$TESTNET_RELAY_CONTRACT_ADDRESS':' .my_env_file.env
+rm .my_env_file.env.OLD
+
+cd ../..
+
+printf "${DONE_START}relay initialized successfully!${DONE_END}"
+
+printf "${LOG_START}Starting relay...${LOG_END}"
+
+# Install python virtualenv.
+pipenv install --python=$(pyenv which python3.7)
```

Comment:
We're missing installation of `python3.7` in the setup script, aren't we?
```
✗ ./run-testnet-relay.sh 

Initializing relay...

TestnetRelay contract address is: 0x022564879D3b20b9Aa6989934482139a819458E7

relay initialized successfully!


Starting relay...
pyenv: python3.7: command not found
Creating a virtualenv for this project...
Pipfile: /Users/jakub/workspace/keep-network/local-setup/relays/maintainer/Pipfile
Using /usr/bin/python3 (3.7.3) to create virtualenv...
⠦ Creating virtual environment...Using base prefix '/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7'
New python executable in /Users/jakub/.local/share/virtualenvs/maintainer-YcemNO6_/bin/python3
Also creating executable in /Users/jakub/.local/share/virtualenvs/maintainer-YcemNO6_/bin/python
Running virtualenv with interpreter /usr/bin/python3

✘ Failed creating virtual environment 
[pipenv.exceptions.VirtualenvCreationException]:   File "/usr/local/lib/python2.7/site-packages/pipenv/cli/command.py", line 254, in install
[pipenv.exceptions.VirtualenvCreationException]:       editable_packages=state.installstate.editables,
[pipenv.exceptions.VirtualenvCreationException]:   File "/usr/local/lib/python2.7/site-packages/pipenv/core.py", line 1741, in do_install
[pipenv.exceptions.VirtualenvCreationException]:       pypi_mirror=pypi_mirror,
[pipenv.exceptions.VirtualenvCreationException]:   File "/usr/local/lib/python2.7/site-packages/pipenv/core.py", line 574, in ensure_project
[pipenv.exceptions.VirtualenvCreationException]:       pypi_mirror=pypi_mirror,
[pipenv.exceptions.VirtualenvCreationException]:   File "/usr/local/lib/python2.7/site-packages/pipenv/core.py", line 506, in ensure_virtualenv
[pipenv.exceptions.VirtualenvCreationException]:       python=python, site_packages=site_packages, pypi_mirror=pypi_mirror
[pipenv.exceptions.VirtualenvCreationException]:   File "/usr/local/lib/python2.7/site-packages/pipenv/core.py", line 935, in do_create_virtualenv
[pipenv.exceptions.VirtualenvCreationException]:       extra=[crayons.blue("{0}".format(c.err)),]
[pipenv.exceptions.VirtualenvCreationException]: Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/virtualenv.py", line 2567, in <module>
    main()
  File "/usr/local/lib/python2.7/site-packages/virtualenv.py", line 793, in main
    symlink=options.symlink,
  File "/usr/local/lib/python2.7/site-packages/virtualenv.py", line 1071, in create_environment
    install_python(home_dir, lib_dir, inc_dir, bin_dir, site_packages=site_packages, clear=clear, symlink=symlink)
  File "/usr/local/lib/python2.7/site-packages/virtualenv.py", line 1552, in install_python
    shutil.copy(original_python, py_executable)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/shutil.py", line 245, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/shutil.py", line 103, in copyfile
    if _samefile(src, dst):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/shutil.py", line 88, in _samefile
    return os.path.samefile(src, dst)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/genericpath.py", line 96, in samefile
    s1 = os.stat(f1)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType

Failed to create virtual environment.
```

Reply:
Maybe we should do `pyenv install 3.7.7`?

Reply:
Yep agree.